packages/aws-cdk/test/api/hotswap/lambda-functions-hotswap-deployments.test.ts

```diff
@@ -37,7 +38,7 @@ test('returns undefined when a new Lambda function is added to the Stack', async
   expect(deployStackResult).toBeUndefined();
 });
 
-test('calls the updateLambdaCode() API when it receives only a code difference in a Lambda function', async () => {
+test('calls the updateLambdaCode() API when it receives only a code difference in a Lambda function (S3 code)', async () => {
```

Comment:
Can you actually move the Docker image tests to a separate file? Same way we have today `lambda-functions-hotswap-deployments.test.ts` and `lambda-versions-aliases-hotswap-deployments.test`? I'm a little worried these tests files are getting a little big.

Maybe the new file can be called `lambda-functions-docker-hotswap-deployments.test.ts`? I would be even fine renaming `lambda-functions-hotswap-deployments.test.ts` to `lambda-functions-s3-hotswap-deployments.test.ts`.